code/datums/jobs/job/xenomorph.dm

```diff
@@ -11,6 +11,10 @@
 /datum/job/xenomorph/return_spawn_type(datum/preferences/prefs)
 	return /mob/living/carbon/xenomorph/larva
 
+/datum/job/xenomorph/add_job_points(amount, origin = MARINE_SPAWN_ORIGIN)
+	. = ..(amount)
+	SSblackbox.record_feedback("tally", "round_statistics", 0.125, origin)
```

Comment:
shoulnd't it  be amount instead of 0.125 ?

Reply:
should be amount * 0.125, me stoopid